drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/enrollment_attribute_calculations.rb

```diff
@@ -116,66 +116,48 @@ def race_list(enrollment)
       end
 
       def ce_infos_for_batch(filter, batch)
-        household_enrollments = household_enrollments(filter, batch)
-        ce_events = ce_events(filter, household_enrollments)
+        # Find the active enrollments in the CE Project set for the HoH of the enrollments in the batch
+        ce_project_enrollments = GrdaWarehouse::ServiceHistoryEnrollment.
+          entry.
+          where(project_id: GrdaWarehouse::Hud::Project.where(id: filter.secondary_project_ids).pluck(:project_id)).
+          open_between(start_date: filter.start_date, end_date: filter.end_date).
+          where(id: batch.joins(:service_history_enrollment_for_head_of_household).select(:id))
```

Comment:
This will only work if the CE (secondary_project_ids) are also in the primary project ids, is that the intention?